### PR TITLE
better error message for vhdr ascii

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

**Comment by Eric89GXL** [comment]
LGTM although adding non-binary data might be a trivial `np.loadtxt` call. Is it more difficult?

**Comment by agramfort** [comment]
I did the best I could with my limited time :)

let's merge this and open an issue with sample ascii data for future work.

ok?

File: mne/io/brainvision/brainvision.py [hunk 1]
```diff
@@ -365,7 +365,10 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
     info = _empty_info(sfreq)
 
     # check binary format
-    assert cfg.get('Common Infos', 'DataFormat') == 'BINARY'
+    data_format = cfg.get('Common Infos', 'DataFormat')
+    if data_format != 'BINARY':
+        raise ValueError('Only support for data stored in binary format '
+                         'is supported. Your data are in %s.' % data_format)
```
Comment (jona-sassenhagen) [inline mne/io/brainvision/brainvision.py]: You mean something like

'Only data in binary format is supported. Your data are in %s.'

?

**Comment by agramfort** [comment]
comment addressed

**Comment by jaeilepp** [comment]
Thanks

**Pull request merged**